hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerCheck.java

```diff
@@ -398,9 +396,7 @@ private void loadContainerData() throws IOException {
   }
 
   private void handleCorruption(IOException e) {
-    String errStr =
-        "Corruption detected in container: [" + containerID + "] ";
-    String logMessage = errStr + "Exception: [" + e.getMessage() + "]";
-    LOG.error(logMessage);
+    LOG.error("Marking Container [{}] UNHEALTHY as it failed metadata check." +
+                    " Exception: {}", containerID, e);
```

Comment:
```suggestion
    LOG.error("Corruption detected in container [{}]. Marking it UNHEALTHY.", containerID, e);
```

Technically the metadata or the data (the actual blocks) may have been corrupted when this method is called. The exception should tell us which one it is. Also we can let the logger handle the exception printing directly without needing to pass it as a string format arg.

Reply:
ohh i see~ ok! Yeah will update the string format!!